addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -318,6 +331,9 @@ QUnit.module('web_editor', {}, function () {
                     if (route.indexOf('/web_unsplash/fetch_images') === 0) {
                         return Promise.resolve();
                     }
+                    if (route.indexOf('/web_editor/media_library_url') === 0) {
+                        return Promise.resolve('http://localhost/');
```

Comment:
I just removed this because we changed the library querying to be s2s, but before this it had to be a valid URL (doesn't matter if it's unreachable) but this is no longer relevant as I removed it.